README.md

```diff
@@ -1 +1,39 @@
 # bridge-workers
+
+### Prerequisites
+
+* Install Docker.
+* Two files on the host, `.aws/credentials` and `.bridge/workers.conf`, are shared with the workers container.
+  * Make sure the default AWS credentials profile in `.aws/credentials` has the correct credentials.
+  * Make sure `.bridge/workers.conf` has the `bridge.user` entry and is set to your user name.
```

Comment:
Any examples of the properties that should be in these files, or dummy sample conf files?